src/components/Login/JellyfinLogin.tsx

```diff
@@ -138,7 +139,9 @@ const JellyfinLogin: React.FC<JellyfinLoginProps> = ({
                         id="username"
                         name="username"
                         type="text"
-                        placeholder={intl.formatMessage(messages.username)}
+                        placeholder={`${intl.formatMessage(
+                          messages.email
+                        )} / ${intl.formatMessage(messages.username)}`}
```

Comment:
@fallenbagel do you think we just just display "Username / Email Address" for Emby users?
I don't want Jellyfin users to get the wrong login and use their email instead of their username.

Reply:
We have a similar placeholder on local login, which already causes tons of confusion amongst users (though it's not as heavily used as jellyfin login, so we can let it be for now).

But this would create a lot of confusion. I don't think this change would be a good idea.

Reply:
Thats fine ig. Though influx of confusion will come from those emby users not using emby connect